conf/app.conf

```diff
@@ -3,3 +3,4 @@ httpaddr = ${APP_HOST||0.0.0.0}
 httpport = ${PORT||8080}
 runmode = ${APP_RUN_MODE||dev}
 db_url = ${DATABASE_URL}
+SessionOn = true
```

Comment:
Added this to [activate the session module](https://beego.me/docs/mvc/controller/session.md#session-control)